Style/index.md

```diff
@@ -14,26 +14,27 @@ A good and short read about the importance of following a chosen style is: [Brow
 
 ## Language Standards and Tools
 
-In order to build NJOY21 from source, support for the following language standards is required:
+In order to build NJOY from source, support for the following language standards is required:
 
  - [Fortran 2003 ](http://j3-fortran.org)
```

Comment:
tiny bit of Fortran 2008 standard in that we use the `newunit` keyword to prevent io clobbering.
reference: http://fortranwiki.org/fortran/show/newunit